### ISSUE TITLE
HardwareVerify - Minor documentation changes required to clarify use

### ISSUE THREAD

**Issue by NigelTwo** [TomNisbet/TommyPROM#13]
The background to this is that I have built the TommyPROM hardware using 74HC595 shift registers. So with both hardware and software changes a bit of testing is needed! As a result I have discovered that the Troubleshooting documentation is misleading:
1. Testing the address lines - the table shows the commands A0, A1 etc to setup the address latches. But the correct commands are A000, A0001 etc. So all four hex digits are required as the program menu shows. See below for some examples of this.
2. Data test commands - look like they have the same problem. Always use 2 hex digits.
3. Testing the control lines - the 4th line in the table has a typo?  "Cd" should read "Od".

To illustrate the address line testing commands I added a print statement to the 'a' command processing code viz
`case 'a':

        if (hexDigit(line[1]) <= 15)
        {
            w = hexWord(line + 1);
            printWord( w );
            Serial.println();
            Serial.println( w, BIN );
            prom.setAddress(w);
        }
        else
            cmdError = true;
        break;`
Now the difference between A0 and A000, and A400 and A0400 can be seen.
`#a0
ffff
1111111111111111

#a0000
0000
0

#a400
40ff
100000011111111

#a0400
0400
10000000000

#`

IMHO the software bug (buffer over-read) doesn't need to be fixed. It is hardware testing code after all. Just adjust the documentation. It will have the side effect of reducing the confusion between A0 the command and A0 the 0th address line, and D0 the command and D0 the 0th data line.

Thanks for taking the time to design and document this project. It's encouraging to see proof that it will work in the end ;-)

**Comment by NigelTwo** [comment]
@TomNisbet - I'll leave you to choose whether to port the changes into the Hardware Tool UI. The A0/D0 confusion is actually a triple context problem that does take a while to get one's head around:
1. the UI commands;
2. the (E)EPROM address lines;
3. the Arduino IDE pin defines (bless them, they meant well).

I am thinking that a new issue might be best for a discussion around the alternative shift register circuits. 

Thanks for your offer of help, but I am pretty certain my variant of the circuit is working now. It is slower, and you have stressed the importance of speed wrt the EEPROM unlocking operations. What a surprise to find timestamps already in the code just about where I was to add them!

**Comment by TomNisbet** [comment]
I checked in a fix so that the CLI parser matches what I thought it was already doing.  The documentation has been updated to clarify the distinction between the D5 command, the D5 data pin on the ROM, and the D5 output on the Arduino.

@NigelTwo - I also added code and documentation to support a 74LS595 variant of the hardware.  The RCLK pins on the shift registers are controlled by the D13 output on the Arduino.  The support needs to be enabled with an edit to the Configure.h file.  Details are in the Hardware page of the documentation.
